consultant.gemspec

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+Gem::Specification.new do |s|
+  s.name        = 'consultant'
+  s.version     = '0.0.0'
+  s.date        = '2020-11-24'
+  s.summary     = "A suite of tools for pinpointing inefficiencies\n"\
+    'and measuring and presenting performance improvements'
+  s.description = 'Description TBD'
+  s.authors     = ['Chris Ruenes, Eitan Bar-David, Noah Levin']
+  s.email       = 'ruenes.chris@gmail.com'
+  s.files       = [
```

Comment:
I saw in a tutorial that we need to list all files that we add, but I'm not 100% sure that's true. When we have some time, we should verify and remove this long list if not necessary